pkg/ipam-node/controllers/cidrpool/cidrpool.go

```diff
@@ -72,7 +72,7 @@ func (r *CIDRPoolReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 			}
 			_, nodeSubnet, _ := net.ParseCIDR(alloc.Prefix)
 			startIP := ip.NextIP(nodeSubnet.IP)
-			if ip.IsPointToPointSubnet(nodeSubnet) {
+			if ip.IsPointToPointSubnet(nodeSubnet) || ip.IsSingleIPSubnet(nodeSubnet) {
```

Comment:
looks like that we eventually may need some more testing on this package?

Reply:
agree, I added some tests to the package